utilities/install-collector.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# netdata
+# real-time performance and health monitoring, done right!
+# (C) 2022
+
+# This script installs community maintained collectors from netdata/community github repo
+
+NETDATA_DIR="/etc/netdata"
```

Comment:
Probably we need to create this script at `netdata/netdata`, because our `kickstart-startic` installs code at `/opt/`, so the default directory is `/opt/etc/netdata`, and as you can see in next code

https://github.com/netdata/netdata/blob/b8d50ecc953f7962587aaae1a793827faf4ce992/netdata-installer.sh#L479-L485

we add a prefix for path.

Reply:
@thiagoftsm ideally, this script should be available on the agent node so users can just run it to install community collectors. Maybe once the review of this is done we can agree on where to move it in `netdata/netdata`?

Reply:
All right, I suggest we use either [packaging](https://github.com/netdata/netdata/tree/master/packaging) or [collectors](https://github.com/netdata/netdata/tree/master/collectors). (ping @Ferroin )